<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportTypePage</class>
 <widget class="QWizardPage" name="ImportTypePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Import type</string>
  </property>
  <property name="subTitle">
   <string>Please select which type of log files you would like to import.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioButtonLocal">
     <property name="text">
      <string>Local: Typically found under /tmp/testfw_log/tests</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonRemote">
     <property name="text">
      <string>Remote: Typically found under /log or /log2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="FileSelectionWidget" name="fileSelectionWidget" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>209</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelectionWidget</class>
   <extends>QWidget</extends>
   <header>widgets/fileselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
